.pre-commit-config.yaml

```diff
@@ -13,6 +13,7 @@ repos:
   rev: v2.7.0
   hooks:
     - id: pyproject-fmt
+      args: ["--max-supported-python=3.14"]
```

Comment:
This is needed because default value is `3.13` and causes the deletion of classifier for Python 3.14.

See https://github.com/tox-dev/toml-fmt/issues/91